lib/x11utils.pm

```diff
@@ -86,6 +88,12 @@ sub ensure_unlocked_desktop {
                 select_user_gnome($username);
             }
         }
+        if (match_has_tag('guest-disabled-display')) {
```

Comment:
@ggardet this correspond to the needle that you created recently, but as screenlock, I'll submit a pr later to do the respective change there. I hope the intention here is clear. 

I'll submit later also a VR for opensuse (In hopes that it behaves the same way too)

Reply:
I doubt that hitting the `shift` key brings the screen back. Does it?

Reply:
According to @SergioAtSUSE it does (And in general, looking at the test runs, it seems to do, I'll trigger a test later and see manually)

Reply:
So, this is definitely not https://bugzilla.opensuse.org/show_bug.cgi?id=1168992